stage/execution_stage.py

```diff
@@ -77,19 +77,19 @@ def execute(self, result_history: list[Result]) -> Result:
       raise TypeError
 
     try:
-      build_result, run_result = evaluator.builder_runner.build_and_run(
+      _, run_result = evaluator.builder_runner.build_and_run(
```

Comment:
A favour to ask:
Could you please keep `build_result` for now?
I know it's not used and may give lint warnings, but sometimes I saw fuzz targets that can compile in `build_result` failed to compile in execution stage.
Keeping `build_result` here helps me debug, and I will remove it once done.